x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java

```diff
@@ -80,6 +80,7 @@ public class DownsampleActionIT extends ESRestTestCase {
                         },
                         "routing_path": ["metricset"],
                         "mode": "time_series",
+                        "look_ahead_time": "1m",
```

Comment:
Hm, is this needed given we configure the start/end time explicitly in the index template to 2021 ?  
``` 
 Strings.format(TEMPLATE, dataStream, "2006-01-08T23:40:53.384Z", "2021-01-08T23:40:53.384Z", policy)
```

The look ahead time would only take effect for the 2nd generation index but we're not writing to it.

Reply:
But I think the end_time can still be still after the data stream creation? (by `UpdateTimeSeriesRangeService`)
It depends on when it happens, by default it runs every 5 minutes. I think it is possible that the test cluster runs for longer than 5 minutes (many relatively heavy tests in the various test classes)? Especially on slower CI?

Reply:
Ah, you're right. `UpdateTimeSeriesRangeService` might recompute the `end_time` using:
```
now.plus(lookAheadTime.getMillis()
```
This makes sense